elyra/pipeline/component.py

```diff
@@ -101,6 +108,14 @@ def control(self) -> str:
     def control_id(self) -> str:
         return self._control_id
 
+    @property
+    def control_types(self) -> Dict[str, str]:
```

Comment:
Could you please add help-strings, particularly for these control-oriented properties, to help clarify things?  What you have for the parameter descriptions is probably sufficient.

Reply:
done